src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1747,17 +1738,29 @@ public void testInputAnnotationScopeIndentationCheck() throws Exception {
         verifyWarns(checkConfig, fileName, expected);
     }
 
-    @Override
-    protected String internalGetCheckMessage(
-            String messageBundle, String messageKey, Object... arguments) {
-        final ResourceBundle resourceBundle = ResourceBundle.getBundle(
-                messageBundle,
-                Locale.ROOT,
-                Thread.currentThread().getContextClassLoader(),
-                new LocalizedMessage.Utf8Control());
-        final String pattern = resourceBundle.getString(messageKey);
-        final MessageFormat formatter = new MessageFormat(pattern, Locale.ROOT);
-        return formatter.format(arguments);
+    @Test
+    public void testPropertyOrderOfErrorMessages() {
+        final int propertyCount = 3;
+        final String[] properties = IntStream.range(0, propertyCount)
+                .mapToObj(index -> "##" + index + "##").toArray(String[]::new);
```

Comment:
Done.
I used new String[] { "##0##", .... } when first writing the test. When doing the review myself, I rewrite the code into this. You are right. It seems really redundant. My mind maybe not very clear when I wrote this.